src/components/TxData.js

```diff
@@ -197,13 +197,21 @@ class TxData extends React.Component {
       }
     }
 
-    const renderOutput = (output, idx, addBadge) => {
+    const renderOutputLink = (idx) => {
+      if (idx in this.props.spentOutputs) {
+        return <span> (<Link to={`/transaction/${this.props.spentOutputs[idx]}`}>Spent</Link>)</span>;
+      } else {
+        return null;
+      }
+    }
+
+    const renderOutput = (output, idx, isOutput) => {
```

Comment:
Also change the method name? `renderOutput` with something that is not an output is confusing

Reply:
I agree that it is confusing. But, in this case, we use the same method because we are rendering "the output that the inputs points to". Maybe we should just add a note about it.

Reply:
Maybe change to `renderInputOrOutput`?

Reply:
Done

Reply:
Looks good